lib/active_fedora/solr_instance_loader.rb

```diff
@@ -27,7 +27,12 @@ def object
     private
 
     def allocate_object
-      active_fedora_class.allocate.init_with_json(profile_json)
+      active_fedora_class.allocate.init_with_json(profile_json) do |allocated_object|
+        create_key = Solrizer.solr_name('system_create', :stored_sortable, type: :date)
```

Comment:
This code (to get the key) is already in Querying. Can we extract it to a constant?